### PR TITLE
added a convoluted neural net

### PR DESCRIPTION
**Pull request by acue**
@marcusmolchany lemme know what you think

### CODE REVIEW

**Comment by marcusmolchany** [comment]
smh

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # jquery.filterjitsu.js
 
-[![Build Status](https://travis-ci.org/Fullscreen/filterjitsu.svg?branch=master)](https://travis-ci.org/Fullscreen/filterjitsu)
+  [![Build Status](https://travis-ci.org/Fullscreen/filterjitsu.svg?branch=master)](https://travis-ci.org/Fullscreen/filterjitsu)
```
Comment (devspinn) [inline README.md]: 🍼💦

**Comment by devspinn** [comment]
this is cool, but if you took even a second to familiarize yourself with the project you'd know that all commit messages are present tense. your flippancy will not get you very far

**Comment by CodyKolodziejzyk** [comment]
@devspinn wow punctuation much?

**Pull request closed without merging**